api/node/routes.go

```diff
@@ -27,15 +27,21 @@ const (
 	peerInfoPath        = "/peerinfo"
 )
 
+// AccStateCheckpointsKey is used as a key for the number of account state checkpoints in the api response
+const AccStateCheckpointsKey = "num_accounts_state_checkpoints"
```

Comment:
why exported?

Reply:
To be able to test it more easily.

Reply:
at least change it so it will has the `erd_` prefix like the rest of metrics